scene/resources/packed_scene.cpp

```diff
@@ -600,6 +634,20 @@ Error SceneState::_parse_node(Node *p_owner, Node *p_node, int p_parent_idx, Has
 			if (ures.is_null()) {
 				value = missing_resource_properties[E.name];
 			}
+		} else if (E.type == Variant::ARRAY && E.hint == PROPERTY_HINT_TYPE_STRING) {
+			int hint_subtype_separator = E.hint_string.find(":");
+			if (hint_subtype_separator >= 0) {
+				String subtype_string = E.hint_string.substr(0, hint_subtype_separator);
+				int slash_pos = subtype_string.find("/");
+				PropertyHint subtype_hint = PropertyHint::PROPERTY_HINT_NONE;
+				if (slash_pos >= 0) {
+					subtype_hint = PropertyHint(subtype_string.substr(slash_pos + 1, subtype_string.size() - slash_pos - 1).to_int());
```

Comment:
```suggestion
					subtype_hint = PropertyHint(subtype_string.get_slice("/", 1).to_int());
```
The whole code can be simplified using `get_slice()` or `get_slice_count()`.